packages/app-mobile/web/public/index.css

```
@@ -9,6 +9,44 @@ body, :root {
 	overflow: hidden;
 }
 
+/* Dialog styling */
+
+dialog.dialog-modal-layer {
+	width: 100%;
+	height: 100%;
+	max-width: unset;
+	max-height: unset;
+
+	margin: 0;
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)